meritocracy/backend/postgres/tables/api.component_taxonomy.sql

```diff
@@ -10,3 +10,4 @@ CREATE TABLE api.component_taxonomy (
 );
 
 GRANT SELECT ON api.component_taxonomy TO web_anon;
+GRANT ALL ON api.component_taxonomy to editor;
```

Comment:
All? maybe read and update should be sufficient. Delete and drop table is to wide

Reply:
well you need `delete` so that you may remove reference from at component to a taxonomy, but I agree that `drop` is to much. I guess this is better:
```suggestion
GRANT SELECT, INSERT, UPDATE, DELETE ON api.component_taxonomy to editor;
```